common/plugins/eu.esdihumboldt.hale.common.align/help/merge/help.xhtml

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
+<title>Merge</title>
+</head>
+<body>
+<div id ="help">
+	<h4>Example mapping:</h4>
+	<p>
+		The merge function is used to merge multiple source features within a single, source dataset. The merged source features create a single instance in the target dataset. In the merge configuration you must specify which instances should be merged together by specifying a merge key property. Instances that have the same value for this key property will be merged into one instance. For the other properties, the values are accumulated in the merged instance.<br>
+	</br>
+		<img src="../../eu.esdihumboldt.hale.common.align/help/merge/mergeMapping.png" /><br>
+	</br>
+		The auto-detect properties function checks for properties where all instances to merge have the same value, and will add only that unique value to the merged instance.<br>
```

Comment:
Using the auto-detect functionality is associated to performance problems right now (see https://github.com/halestudio/hale/issues/626)
Should we document that here?
I don't remember if there was any action taken on that ticket (e.g. adding a warning not to use it)